site/layouts/assessment_wrote_en-en.hbs

```diff
@@ -0,0 +1,202 @@
+				<template>
+					<li><span data-about></span> (CSS selector: <code data-css>Not applicable</code>)</li>
+				</template>
+			</ul>
+			<p>Suggested list of expertise topics that might need to be involved in the subject assessment in order to produce a complete accessibility conformance report. Note: the following list may be empty if not applicable.</p>
+			<ul data-wb-json='{
+					"url": "#[a11y]/acr:involvesExpertise",
+					"mapping": { "selector": "[data-expertise]" }
+				}'>
+				<template>
+					<li><span data-expertise></span></li>
+				</template>
+			</ul>
+		</details>
+	</dd>
+	<dt>Toward a conformity at:</dt>
+	<dd><span data-json-replace="#[a11y]/acr:standard">Not specified</span> (Option: <span data-json-replace="#[a11y]/acr:conformanceOption">Not specified</span>)</dd>
+	<dt>Summary:</dt>
+	<dd data-json-replace="#[a11y]/dct:description">No note</dd>
+</dl>
+
+<h2 id="acr">Assessment details</h2>
+<table class="table">
+	<thead>
+		<tr>
+			<th class="col-md-3">Success Criteria</th>
+			<th class="col-md-2">Result</th>
+			<th>Comment</th>
+		</tr>
+	</thead>
+	<tbody data-wb-json='{
+		"url": "#[a11y]/earl:result",
+		"tobeclone": "tr",
+		"source": "#tmpl-result",
+		"mapping": [
+			{ "selector": "[data-label]", "value": "/earl:test" },
+			{ "selector": "[data-outcome]", "value": "/earl:outcome" },
+			{ "selector": "[data-applicability]", "value": "/earl:subject/dct:description" },
+			{ "selector": "[data-notes]", "value": "/dct:description" },
+			{ "selector": "[data-mode]", "value": "/earl:mode" },
+			{ "selector": "[data-severity]", "value": "/acr:severity" },
+			{ "selector": "[data-relevancy]", "value": "/acr:relevancy" },
+			{
+				"selector": "[data-specific-results]",
+				"value": "/earl:result",
+				"mapping": [
+					{ "selector": "[data-result-label]", "value": "/earl:test" },
+					{ "selector": "[data-result-outcome]", "value": "/earl:outcome" },
+					{ "selector": "[data-result-severity]", "value": "/acr:severity" },
+					{ "selector": "[data-result-relevancy]", "value": "/acr:relevancy" },
+					{ "selector": "[data-result-notes", "value": "/dct:description" }
+				]
+			}
+		]
+	}'>
+	</tbody>
+</table>
+
+<template id="tmpl-result">
+	<table>
+		<tr>
+			<th data-label></th>
+			<td><span data-outcome></span></td>
+			<td>
+				<p><strong>Applicability</strong>: <span data-applicability>Not re-specified</span></p>
+				<ul class="list-inline clearfix">
+					<li>Test mode: <span data-mode>Unknown</span></li>
+					<li class="pull-right clr-rght-md clr-rght-lg">Severity: <span data-severity>Not specified</span></li>
+					<li class="pull-right clr-rght-md clr-rght-lg">Relevance: <span data-relevancy>Not specified</span></li>
+				</ul>
+				<p data-notes></p>
+				<dl data-specific-results>
+					<template>
+						<dt data-result-label></dt>
+						<dd>( <em data-result-outcome></em> <span data-result-severity></span> <span data-result-relevancy></span> ) <span data-result-notes></span></dd>
```

Comment:
Unnecessary space:
```suggestion
						<dd>( <em data-result-outcome></em> <span data-result-severity></span> <span data-result-relevancy></span>) <span data-result-notes></span></dd>
```

Reply:
Applies to French also

Reply:
I added a space to have a consistent rendering because we can't do any conditional with data-json yet. Except for the outcome, the other property might wont be set except for advisory technique.

Reply:
Right, I get it.